content/support/faqs/overview@ko.md

```diff
@@ -0,0 +1,75 @@
+---
+$title@: AMP 살펴보기
```

Comment:
Please remove the `@` for any labels that have translated text in these support/faq files.   For example:

- change `title@:` to `title:`
- change `answer@:` to `answer:`
- change `link_text@:` to `link_text:`
- change `description@:` to `description:`